src/components/vanilla/controls/Dropdown/index.tsx

```diff
@@ -126,6 +126,10 @@ export default (props: Props) => {
           value={search}
           name="dropdown"
           placeholder={props.placeholder}
+          onClick={() => {
```

Comment:
@cwbuecheler If i'm not wrong you can combine onClick and onFocus, placing both setFocus and setTriggerBlur inside onFocus. That way, you no longer need onClick, since clicking the input already triggers onFocus, avoiding redundant calls to setFocus.